utils/index.js

```diff
@@ -1,9 +1,7 @@
 import * as keycodes from './keycodes';
-import * as nodetypes from './nodetypes';
```

Comment:
Good! I like the imports from `window.Node` better. One less piece of duplication